packages/chakra-ui-docs/utils/index.js

```diff
@@ -18,11 +20,33 @@ export const removeHyphenFromString = (hyphenatedString) => {
   return str.join(' ')
 }
 
-export const findNextAndPrevRoute = (path, routes) => {
-  const currentRouteIndex = routes.map(route => route.path).indexOf(path)
+export const findNextAndPrevRoute = (path) => {
+  const orderedRoutes = [...topNavLinks, ...aboutNavLinks, ...components]
+
+  const extractedRoutes = []
+  orderedRoutes.forEach((singleRoute) => {
+    extractedRoutes.push({ name: singleRoute, path: stringToUrl(singleRoute) })
+  })
+
+  const currentRouteIndex = extractedRoutes.map(route => route.path).indexOf(path)
+
+  const nextPage = extractedRoutes[currentRouteIndex + 1]
+  const prevPage = extractedRoutes[currentRouteIndex - 1]
 
-  const nextPageLink = routes[currentRouteIndex + 1]
-  const prevPageLink = routes[currentRouteIndex - 1]
+  const prevPageLink = prevPage
+    ? {
+      ...prevPage,
+      name: (prevPage.name =
+          prevPage.name.charAt(0).toUpperCase() + prevPage.name.slice(1))
+    }
+    : ''
+  const nextPageLink = nextPage
+    ? {
+      ...nextPage,
+      name: (nextPage.name =
+          nextPage.name.charAt(0).toUpperCase() + nextPage.name.slice(1))
+    }
+    : ''
 
-  return { prev: prevPageLink ? prevPageLink.path : '', next: nextPageLink ? nextPageLink.path : '' }
+  return { prev: prevPageLink, next: nextPageLink }
```

Comment:
Great work :)

What's the difference between the (`prevPage` and `nextPage`) and (prevPageLink and nextPageLink), seems redundant to me. The reason is cause I commented out the page-link code and use prev-next-page and it works as expected

Reply:
Yeah, good point, I did not need it after implementing the new changes, nice catch 👍🏽